### ISSUE TITLE
Fix backend unexpected path warning

### ISSUE THREAD

**Issue by wilsonianb** [interledger/rafiki#831]
https://github.com/interledger/rafiki/blob/e067823ff1e0a1f9cb13b4b14587c52547db70b1/packages/backend/src/app.ts#L323-L325

Should be updated to also not log for `/jwks.json`

As is, we are getting:
```
{"level":40,"time":1670505964975,"pid":1,"hostname":"16a8e2c32bb7","path":"/jwks.json","method":"get","msg":"unexpected path/method"}
```

**Comment by PatriciaHeimfarth** [comment]
Hi, can I work on that issue?

**Comment by wilsonianb** [comment]
Yes! :rocket:

**Comment by PatriciaHeimfarth** [comment]
In the readme there is a "Peer-to-peer transfer folder of the Postman collection" mentioned, where can I find it?

**Comment by mkurapov** [comment]
@wilsonianb 
This will be fixed by #916, correct?

**Comment by wilsonianb** [comment]
> @wilsonianb This will be fixed by #916, correct?

yes, but that check can also just be changed to
```
!['/', '/jwks.json'].includes(path) || method !== HttpMethod.GET
```

**Comment by mkurapov** [comment]
Closed with #916
